examples/next/src/app/fixtures/experiments/withTogglePoint.ts

```diff
@@ -8,9 +8,9 @@ const getActiveFeatures = () => getFeatures().decisions;
 
 const withTogglePoint = withTogglePointFactory({
   getActiveFeatures,
-  logError: console.log,
   plugins: [
     {
+      onVariantError: console.log,
```

Comment:
Do we wan this instead?
```suggestion
      onVariantError: console.error,
```

Reply:
Yeah good shout!